draft-ietf-rtcweb-mdns-ice-candidates.md

```diff
@@ -705,9 +705,12 @@ Specification Requirements {#requirements}
 ============
 
 The proposal relies on identifying and resolving any mDNS-based ICE candidates
-as part of adding/processing a remote candidate.
-{{ICESDP}} section 4.1 could be updated to explicitly allow mDNS names in the
-connection-address field.
+as part of adding/processing a remote candidate. {{ICESDP}} section 4.1 could
```

Comment:
This section is going away. I suggest putting this at the end of 3.1 instead (not that 4.1 should be updated, but that when the mDNS candidate is the default candidate, the FQDN should be inserted into the c= line like any other candidate).

Also think we should take a more specific position on the IP4/IP6 family choice; I recommend saying that implementations SHOULD use IP4 in all cases.

Reply:
> Also think we should take a more specific position on the IP4/IP6 family choice; I recommend saying that implementations SHOULD use IP4 in all cases.

I am not quite sure how this c= line is used in practice.
I am fine with IP4 as a default. If the device is IPv6 only, shouldn't it be IP6 though?

Reply:
Updated and PTAL.